packages/data-stores/src/plans/selectors.ts

```diff
@@ -22,21 +32,37 @@ export const getPlanBySlug = ( state: State, slug: PlanSlug ): Plan => {
 	return state.plans[ slug ] ?? undefined;
 };
 
-export const getDefaultPaidPlan = ( _: State, locale: string ): Plan => {
-	return select( STORE_KEY ).getPlansDetails( locale )?.plans[ DEFAULT_PAID_PLAN ];
+export const getDefaultPaidPlan = (
+	_: State,
+	locale: string,
+	billPeriod: 'ANNUALLY' | 'MONTHLY' = 'ANNUALLY'
```

Comment:
Should we create a new `PlanBillingPeriod` type in `packages/data-stores/src/plans/types.ts` and use it throughout the store?